charts/apisix/README.md

```diff
@@ -141,6 +141,14 @@ If etcd.enabled is true, set more values of bitnami/etcd helm chart use etcd as
 
 Default enabled plugins. See [configmap template](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/configmap.yaml) for details.
 
+
+### external plugin parameters
+
+| Parameter                       | Description                                                                                                                                                      | Default                     |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| `extPlugin.enabled` | Whether to allow an ext-plugin | `false` |
```

Comment:
I prefer to add the hyperlink of external plugins: https://apisix.apache.org/docs/apisix/next/external-plugin/

Reply:
Happy to add a hyperlink here